addons/nuxeo-retention/nuxeo-retention-web-ui/src/main/resources/web/nuxeo.war/ui/search/retention/nuxeo-retention-search-form.html

```diff
@@ -50,7 +50,7 @@
                    value="{{searchTerm}}"
                    placeholder="[[i18n('defaultSearch.fullText.placeholder')]]"
                    autofocus
-                   on-keydown="_checkForEnter">
```

Comment:
Not sure we want to change this behaviour. Was it a requirement?

Reply:
see my comment: https://github.com/nuxeo/nuxeo/pull/3907#issue-401422309.

Reply:
Thanks!
As discussed we can keep the `_checkForEnter` and add `auto` property to our `retentionSearch` contribution on `nuxeo-retention.html`